### PR TITLE
delete repetitive tests [changelog skip]

### PR DESCRIPTION
**Pull request by lillianzhang331**
Delete tests that are repeated

### CODE REVIEW

File: test/run [hunk 1]
```diff
@@ -456,10 +456,6 @@ testWarnDevDeps() {
   assertCaptured "A module may be missing"
   assertNotCaptured "This module may be specified"
   assertCapturedError
-  compile "failing-build"
-  assertNotCaptured "A module may be missing"
-  assertNotCaptured "This module may be specified"
-  assertCapturedError
```
Comment (lillianzhang331) [inline test/run:462]: failing-build is already being compiled elsewhere and tested for elsewhere

File: test/run [hunk 2]
```diff
@@ -794,14 +789,6 @@ testFailingBuild() {
   assertCapturedError 1 ""
 }
 
-testTicketOnFailure() {
-  compile "invalid-dependency"
-  assertCaptured "troubleshooting-node-deploys"
-  assertCaptured "please submit a ticket"
-  assertNotCaptured "possible problems"
-  assertCapturedError
-}
```
Comment (lillianzhang331) [inline test/run:803]: moved these assertions so that invalid-dependency does not need to be compiled twice

File: test/run [hunk 3]
```diff
@@ -968,24 +958,6 @@ testNodeEnvTestDepenenciesYarn() {
...
-  assertCapturedSuccess
-}
-
-# When NPM_CONFIG_PRODUCTION = true we should not install devDependencies with Yarn
-testOnlyDependenciesInstalledYarn() {
-  env_dir=$(mktmpdir)
-  echo "true" > $env_dir/YARN_PRODUCTION
-  compile "dependencies-yarn" "$(mktmpdir)" $env_dir
-  assertNotCaptured "lodash"
-  assertCapturedSuccess
```
Comment (lillianzhang331) [inline test/run:986]: these two are repetitive tests

File: test/run [hunk 4]
```diff
@@ -1067,12 +1039,6 @@ testCIEnvVars() {
   assertCapturedSuccess
 }
 
-# If compile fails, test-compile should also fail
-testCICompileFails() {
-  testCompile "failing-build"
-  assertCapturedError
-}
-
```
Comment (lillianzhang331) [inline test/run:1075]: asserting error is captured is already checked for when "failing-build" is compiled elsewhere

**Review by danielleadams — APPROVED** [review]

**Pull request merged**